pkg/cloud/aws/services/ec2/bastion.go

```diff
@@ -101,7 +101,7 @@ func (s *Service) DeleteBastion() error {
 func (s *Service) describeBastionInstance() (*v1alpha1.Instance, error) {
 	input := &ec2.DescribeInstancesInput{
 		Filters: []*ec2.Filter{
-			filter.EC2.ProviderRole(tags.ValueBastionRole),
+			filter.EC2.ProviderRole(v1alpha1.ValueBastionRole),
```

Comment:
Same here, this should be prefixed somehow

Reply:
Any suggestions here? I don't think a prefix on the roles makes sense, but happy to change if you feel strongly...maybe `ValueBastionRoleTagValue`?

Reply:
actually on second thought maybe a `Tag` prefix makes sense... but `Value` is already there. my first suggestion was not good

Reply:
actually on second thought maybe a `Tag` prefix makes sense... but `Value` is already there. my first suggestion was not good

Reply:
ah, similar to what I did with the key => `BastionRoleTagValue` move the prefix to a suffix and add the word `Tag`.

Thoughts?

Reply:
+1 from me on `BastionRoleTagValue`

Reply:
+2 `BastionRoleTagValue`